features/dom.yml.dist

```diff
@@ -0,0 +1,352 @@
+  #   firefox: "50"
+  #   firefox_android: "50"
+  #   safari: "10"
+  #   safari_ios: "10"
+  - api.NodeList.forEach
+
+  # baseline: high
+  # baseline_low_date: 2018-04-30
+  # baseline_high_date: 2020-10-30
+  # support:
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "17"
+  #   firefox: "3.5"
+  #   firefox_android: "4"
+  #   safari: "3"
+  #   safari_ios: "3"
+  - api.NodeIterator.pointerBeforeReferenceNode
+  - api.NodeIterator.referenceNode
+
+  # baseline: high
+  # baseline_low_date: 2018-04-30
+  # baseline_high_date: 2020-10-30
+  # support:
+  #   chrome: "29"
+  #   chrome_android: "29"
+  #   edge: "17"
+  #   firefox: "25"
+  #   firefox_android: "25"
+  #   safari: "9"
+  #   safari_ios: "9"
+  - api.CharacterData.nextElementSibling
+  - api.CharacterData.previousElementSibling
+
+  # baseline: high
+  # baseline_low_date: 2018-04-30
+  # baseline_high_date: 2020-10-30
+  # support:
+  #   chrome: "49"
+  #   chrome_android: "49"
+  #   edge: "17"
+  #   firefox: "49"
+  #   firefox_android: "49"
+  #   safari: "10.1"
+  #   safari_ios: "10.3"
+  - api.DOMTokenList.supports
+
+  # baseline: high
+  # baseline_low_date: 2018-04-30
+  # baseline_high_date: 2020-10-30
+  # support:
+  #   chrome: "50"
+  #   chrome_android: "50"
+  #   edge: "17"
+  #   firefox: "47"
+  #   firefox_android: "47"
+  #   safari: "10"
+  #   safari_ios: "10"
+  - api.DOMTokenList.value
+
+  # baseline: high
+  # baseline_low_date: 2018-04-30
+  # baseline_high_date: 2020-10-30
+  # support:
+  #   chrome: "54"
+  #   chrome_android: "54"
+  #   edge: "17"
+  #   firefox: "49"
+  #   firefox_android: "49"
+  #   safari: "10"
+  #   safari_ios: "10"
+  - api.CharacterData.after
+  - api.CharacterData.before
+  - api.CharacterData.replaceWith
+
+  # baseline: high
+  # baseline_low_date: 2018-04-30
+  # baseline_high_date: 2020-10-30
+  # support:
+  #   chrome: "61"
+  #   chrome_android: "61"
+  #   edge: "17"
+  #   firefox: "49"
+  #   firefox_android: "49"
+  #   safari: "10.1"
+  #   safari_ios: "10.3"
+  - api.DOMTokenList.replace
+
+  # baseline: high
+  # baseline_low_date: 2020-01-15
+  # baseline_high_date: 2022-07-15
+  # support:
+  #   chrome: "51"
+  #   chrome_android: "51"
+  #   edge: "79"
+  #   firefox: "36"
+  #   firefox_android: "36"
+  #   safari: "10"
+  #   safari_ios: "10"
+  - api.NodeList.@@iterator
+
+  # baseline: high
+  # baseline_low_date: 2021-11-02
+  # baseline_high_date: 2024-05-02
+  # support:
+  #   chrome: "53"
+  #   chrome_android: "53"
+  #   edge: "79"
+  #   firefox: "94"
+  #   firefox_android: "94"
+  #   safari: "15"
+  #   safari_ios: "15"
+  - api.ShadowRoot.delegatesFocus
+
+  # ⬇️ Same status as overall feature ⬇️
+  # baseline: low
+  # baseline_low_date: 2024-09-16
+  # support:
+  #   chrome: "125"
+  #   chrome_android: "125"
+  #   edge: "125"
+  #   firefox: "128"
+  #   firefox_android: "128"
+  #   safari: "18"
+  #   safari_ios: "18"
+  - api.ShadowRoot.serializable
```

Comment:
a <kbd>ctrl</kbd>+<kbd>f</kbd> for "shadow" will probably reveal a bunch of things that _aren't_ part of some basic DOM stuff.